htdocs/lib/MapLocators.js

```diff
@@ -0,0 +1,208 @@
+//
+// Map Locators Management
+//
+
+LocatorManager.strokeOpacity = 0.8;
```

Comment:
Actually, helps my readability to have all the constants on the top, but ok.

Reply:
Well, it's probably not the most important thing to discuss. I just commented on this because it took me a minute to understand how this code doesn't cause major breakage.